### PR TITLE
Updated CustomItems & CustomRoles give commands to use default RAUtils.ProcessPlayerIdOrNamesList

### PR DESCRIPTION
**Pull request by icedchai**
## Description
**Describe the changes** 

**What is the current behavior?** (You can also link to an open issue here)
Commands currently evaluate only `*`, `all`, or `Player.Id`

**What is the new behavior?** (if this is a feature change)
Commands now evaluate players based on `RAUtils.ProcessPlayerIdOrNamesList` (eg `@player.17.15`) as well as `*`, `all`, or `Player.Id`.

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No breaking changes

**Other information**:

<br />

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [X] I have checked the project can be compiled
- [X] I have tested my changes and it worked as expected

### Patches (if there are any changes related to Harmony patches)
- [X] I have checked no IL patching errors in the console

### Other
- [ ] Still requires more testing

### CODE REVIEW

**Review by louis1706 — CHANGES_REQUESTED** [review]

**Comment by louis1706** [comment]
Also build your branch with Release not Debug because you wouldn't seen all the error to fix issue of compiling

**Comment by icedchai** [comment]
Alright. I'll do that as soon as I can.

**Review by VALERA771 — CHANGES_REQUESTED** [review]
Some foreach loops can be simplified with LINQ extensions and PR must target dev branch

File: EXILED/Exiled.CustomItems/Commands/Give.cs [hunk 1]
```diff
@@ -97,27 +98,44 @@ public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out s
...
+                    break;
+            }
 
-                    if (!CheckEligible(player))
-                    {
-                        response = "Player cannot receive custom items!";
-                        return false;
-                    }
+            string[] newargs;
+            List<ReferenceHub> list = RAUtils.ProcessPlayerIdOrNamesList(arguments, 1, out newargs);
```
Comment (louis1706) [inline EXILED/Exiled.CustomItems/Commands/Give.cs:105]: you could use Player.GetProcessedData(argument..)

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: EXILED/Exiled.CustomItems/Commands/Give.cs (hunk 1 above)
Comment (icedchai) [reply EXILED/Exiled.CustomItems/Commands/Give.cs:105]: how come? is it to avoid creating a var that wont get used?

Comment (louis1706) [reply EXILED/Exiled.CustomItems/Commands/Give.cs:105]: to not need the way to do multiple Player.Get

Comment (louis1706) [reply EXILED/Exiled.CustomItems/Commands/Give.cs:105]: https://github.com/icedchai/EXILED/pull/1

Comment (icedchai) [reply EXILED/Exiled.CustomItems/Commands/Give.cs:105]: okay, makes sense

File: EXILED/EXILED.props [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
   <PropertyGroup>
     <!-- This is the global version and is used for all projects that don't have a version -->
-    <Version Condition="$(Version) == ''">8.14.0</Version>
+    <Version Condition="$(Version) == ''">8.14.1</Version>
```
Comment (louis1706) [inline EXILED/EXILED.props:18]:
```suggestion
    <Version Condition="$(Version) == ''">8.14.0</Version>
```

File: EXILED/EXILED.props [hunk 3]
```diff
@@ -15,7 +15,7 @@
 
   <PropertyGroup>
     <!-- This is the global version and is used for all projects that don't have a version -->
-    <Version Condition="$(Version) == ''">8.14.1</Version>
+    <Version Condition="$(Version) == ''">8.14.0</Version>
```
Comment (louis1706) [inline EXILED/EXILED.props:18]: Why ?

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: EXILED/Exiled.CustomRoles/Commands/Give.cs [hunk 4]
```diff
@@ -96,16 +97,24 @@ public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out s
...
-                        }
-
-                        role.AddRole(ply);
-                        response = $"{role.Name} given to {ply.Nickname}.";
-                        return true;
+                        break;
+                }
+
+                IEnumerable<Player> list = Player.GetProcessedData(arguments, 1);
+                if (list == null)
```
Comment (Misfiy) [inline EXILED/Exiled.CustomRoles/Commands/Give.cs:104]: Same here

**Review by Misfiy — CHANGES_REQUESTED** [review]

**Review by louis1706 — APPROVED** [review]

File: EXILED/Exiled.CustomItems/Commands/Give.cs [hunk 5]
```diff
@@ -18,6 +18,8 @@ namespace Exiled.CustomItems.Commands
     using Exiled.Permissions.Extensions;
 
     using RemoteAdmin;
+    using UnityStandardAssets.Effects;
```
Comment (VALERA771) [inline EXILED/Exiled.CustomItems/Commands/Give.cs:21]: ???

**Review by VALERA771 — APPROVED** [review]
Lgtm

**Review by Misfiy — APPROVED** [review]

**Pull request merged**